src/compiler/checker.ts

```diff
@@ -10902,7 +10904,7 @@ namespace ts {
             return reportedError;
         }
 
-        function *generateJsxAttributes(node: JsxAttributes): ElaborationIterator {

```

Comment:
Having asterisk before/after is base MS's coding standard. I do not exactly know what MS's coding standard is but if this match might be a good fix?